diff --git i/libfprint/meson.build w/libfprint/meson.build
index 3aa29d4..4c37c70 100644
--- i/libfprint/meson.build
+++ w/libfprint/meson.build
@@ -127,6 +127,7 @@ driver_sources = {
         'drivers/fte3600-brisk.c',
         'drivers/fte3600-template.c',
     ),
+    'fte7001' : files('drivers/fte7001.c'),
     'nb1010' : files('drivers/nb1010.c'),
     'virtual_image' : files('drivers/virtual-image.c'),
     'virtual_device' : files('drivers/virtual-device.c'),
diff --git i/meson.build w/meson.build
index 8ed7905..d3c4da8 100644
--- i/meson.build
+++ w/meson.build
@@ -148,9 +148,10 @@ drivers_info = {
     'mafpmoc': {},
     'secugen': {},
 
-    # SPI drivers (fte3600 is optional/experimental)
+    # SPI drivers (fte3600/fte7001 are optional/experimental)
     'elanspi': { 'spi': true, 'helper': ['udev'], 'optional': not have_spi },
     'fte3600': { 'spi': true, 'helper': ['udev', 'gpiod'], 'optional': true },
+    'fte7001': { 'spi': true, 'helper': ['udev', 'gpiod'], 'optional': true },
 
     # Virtual drivers (test-only, optional)
     'virtual_image': { 'virtual': true, 'helper': ['virtual'], 'optional': true },
